example/src/stedi/example.clj

```diff
@@ -4,9 +4,9 @@
 (defn wrap-slurp
   "Example middleware to show off middleware pattern with lambdas."
   [handler]
-  (fn [{:keys [input-stream] :as req}]
+  (fn [{:keys [input] :as req}]
     (let [resp (handler (-> req
-                            (assoc :payload (slurp input-stream))))]
+                            (assoc :payload (slurp input))))]
```

Comment:
Can we bag "payload" here and stick to `:input`?

Reply:
Fixed